test/e2e/issuer.ts

```diff
@@ -3,23 +3,14 @@
 
 import fetch from 'node-fetch'; // using node-fetch to allow usage of client side certificate with a fetch-like API. This is not a standard webcrypto behaviour, therefore using native node fetch does not work.
 import fs from 'node:fs';
+
 import https from 'node:https';
 
-import {
-	IssuerConfig,
-	MediaType,
-	PRIVATE_TOKEN_ISSUER_DIRECTORY,
-	TOKEN_TYPES,
-	TokenChallenge,
-	publicVerif,
-	arbitraryBatched,
-	util,
-} from '@cloudflare/privacypass-ts';
+import { UniversalFetch, UniversalRequestInit, requestBatchedTokens, requestSingleToken } from './e2eUtils.js';
+
 
-const { BlindRSAMode, Client, Origin } = publicVerif;
-const { TokenRequest, Client: BatchedTokensClient, BatchedTokenResponse } = arbitraryBatched;
-import { type Token } from '@cloudflare/privacypass-ts';
-import { b64URLtoB64, b64Tou8 } from '../../src/utils/base64';
+const b64Tou8 = (b: string): Uint8Array => Uint8Array.from(atob(b), c => c.charCodeAt(0));
+const b64URLtoB64 = (s: string): string => s.replace(/-/g, '+').replace(/_/g, '/');
```

Comment:
you should be able to import the ones from https://github.com/cloudflare/privacypass-issuer/blob/main/src/utils/base64.ts